apps/docs/stories/utils/decorators.ts

```diff
@@ -90,3 +91,20 @@ export const getAttributesString = (args: Args) => {
     })
     .join(' ');
 };
+
+
+export const themedStoryDecorator = (storyComponent: any, storyArgs: any) => {
```

Comment:
New decorator - this can slowly replace `customStoryDecorator` and then we can remove it